frontend/src/components/LoginForm.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <form class="box" action="" v-on="submit">
+    <BaseInput
+      id="email"
+      label="Email"
+      v-model="email"
+      type="email"
+      class="login-form-item base-input-field"
+      placeholder="ola@nordman.no"
```

Comment:
This is never rendered, as label is set at placeholder - might as well remove it.

Reply:
:+1: